.github/workflows/cmake.yml

```diff
@@ -42,20 +56,62 @@ jobs:
     - uses: actions/checkout@v3
       with:
         submodules: 'recursive'
+
     - uses: actions/setup-python@v4
       with:
-        python-version: '3.10'
+        python-version: "3.10"
+
     - uses: BSFishy/pip-action@v1
       with:
         requirements: requirements.txt
 
+    - name: check_env
+      run: |
+        which python
+        python -m pip list
+
     - name: Build
-      shell: bash
       # Execute the build.  You can specify a specific target with "--target <NAME>"
-      run: python3 setup.py build -- ${{ matrix.instruction_set_flag }} -DCMAKE_C_COMPILER=${{ matrix.c_compiler }} -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }}
+      run: python setup.py build -- ${{ matrix.instruction_set_flag }} -DCMAKE_C_COMPILER=${{ matrix.c_compiler }} -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }}
```

Comment:
Just normalising the use of `python` here, I don't really think it matters either way, so happy to revert.